plugins/aladino/functions/context.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Explore.dev, Unipessoal Lda - All Rights Reserved
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+package plugins_aladino_functions
+
+import (
+	"github.com/reviewpad/reviewpad/v3/handler"
+	"github.com/reviewpad/reviewpad/v3/lang/aladino"
+)
+
+func Context() *aladino.BuiltInFunction {
+	return &aladino.BuiltInFunction{
+		Type:           aladino.BuildFunctionType([]aladino.Type{}, aladino.BuildStringType()),
+		Code:           contextCode,
+		SupportedKinds: []handler.TargetEntityKind{handler.PullRequest, handler.Issue},
+	}
+}
+
+func contextCode(e aladino.Env, args []aladino.Value) (aladino.Value, error) {
```

Comment:
I think we should use the blank identifier here to state that the variable or argument is being intentionally ignored.

```suggestion
func contextCode(e aladino.Env, _ []aladino.Value) (aladino.Value, error) {
```